CompanyEmployees.Infrastructure.Persistence/RepositoryContext.cs

```diff
@@ -14,8 +14,10 @@ public RepositoryContext(DbContextOptions options)
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
             modelBuilder.ApplyConfiguration(new CompanyConfiguration());
+            modelBuilder.ApplyConfiguration(new AttendanceConfiguration());
```

Comment:
Ensure that `AttendanceConfiguration` is implemented and correctly configures all required properties (e.g., keys, relationships, constraints). Missing or incorrect configuration could lead to runtime errors or incorrect schema generation.

```suggestion
modelBuilder.ApplyConfiguration(new AttendanceConfiguration());
```